rancher-shell-0.4.yaml

```
@@ -0,0 +1,73 @@
+package:
+  name: rancher-shell-0.4
+  description: kubectl and helm installer image for Rancher
+  epoch: 0
+  version: 0.4.1
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - bash
+      - bash-completion
+      - busybox
+      - curl
+      - gnutar
+      - gzip
+      - helm
+      - jq
+      - k9s
+      - kubectl
+      - kustomize
+      - posix-libc-utils
+      - vim
+
+environment:
+  contents:
+    packages:
+      - bash
+      - build-base
+      - busybox
+      - ca-certificates-bundle
```

```suggestion
      - busybox
```
nit: only busybox should be good here.